command/helper/helper.go

```diff
@@ -483,3 +483,19 @@ func FormatKV(in []string) string {
 
 	return columnize.Format(in, columnConf)
 }
+
+// DirectoryExists checks if the directory at the specified path exists
+func DirectoryExists(directoryPath string) bool {
+	// Grab the absolute filepath
+	pathAbs, err := filepath.Abs(directoryPath)
+	if err != nil {
+		return false
+	}
+
+	// Check if the directory exists, and that it's actually a directory if there is a hit
+	if fileInfo, statErr := os.Stat(pathAbs); os.IsNotExist(statErr) || !fileInfo.IsDir() {
+		return false
+	}
+
+	return true
```

Comment:
What happens if we get another error that isn't `not exist` one?
If we get it, and `fileInfo` is nil I think that we're gonna get null pointer exception from `fileInfo.IsDir()`

Reply:
You're right, we're not handling that kind of case.
I checked and fileInfo will indeed be nil if any kind of error occurs that's not `not exists`.

Fixed 👍 
ba7e15e